pkg/privilege/privileges/cache.go

```diff
@@ -2128,12 +2220,19 @@ func (h *Handle) Update(userList []string) error {
 		return nil
 	}
 
+	res, err := h.sctx.Get()
```

Comment:
Actually I do not quite understand why we can skip reload if the user is not 'active'. Should we also Update the user that is not 'active'?